packages/accessibility/src/behaviors/Accordion/accordionTitleBehavior.ts

```diff
@@ -24,7 +24,7 @@ const accordionTitleBehavior: Accessibility<AccordionTitleBehaviorProps> = props
       },
       content: {
         'aria-expanded': !!props.active,
-        'aria-disabled': !!(props.active && !props.canBeCollapsed),
+        'aria-disabled': !!((props.active && !props.canBeCollapsed) || props.disabled),
```

Comment:
I don't how it can be simplified, but I don't understand what it does 😿

Reply:
Same here, couldn't understand in order to propose better condition :(